src/Common/RJSF/rjsfForm.scss

```diff
@@ -1,3 +1,12 @@
+.rjsf-root-css #root {
```

Comment:
instead of any level child, can we do specific level stling?

Reply:
Specifying root is no longer needed as discussed. Also changed the form classname